### PR TITLE
fix: decentraland dependencies

### PR DESCRIPTION
**Pull request by 2fd**

### CODE REVIEW

File: src/eth/provider.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
-import { connection, ConnectionResponse } from 'decentraland-connect'
-import { WebSocketProvider } from 'eth-connect'
+import type { ConnectionResponse, Provider } from 'decentraland-connect/dist/types'
+import { connection } from 'decentraland-connect/dist/ConnectionManager'
+import { ProviderAdapter } from 'decentraland-connect/dist/ProviderAdapter'
+import { WebSocketProvider } from 'eth-connect/providers/WebSocketProvider'
```
Comment (kevinszuchet) [inline src/eth/provider.ts:4]: Could you move this import up, please? It should be before Decentraland-related packages.

File: src/eth/provider.ts [hunk 2]
```diff
@@ -15,15 +16,15 @@ export async function getEthereumProvider(
...
+  provider: Provider
   chainId: number
   account: string | null
 }> {
   if (type === null) {
     const rpc = chainIdRpc.get(chainId)
     if (!rpc) throw new Error("Can't get RPC for chainId " + chainId)
     return {
-      provider: new WebSocketProvider(rpc),
+      provider: new ProviderAdapter(new WebSocketProvider(rpc) as any) as any,
```
Comment (kevinszuchet) [inline src/eth/provider.ts:27]: Why are we using any instead of `Provider`?

File: src/eth/provider.ts (hunk 1 above)
Comment (2fd) [reply src/eth/provider.ts:4]: `eth-connect` is a decentraland-related package

File: src/eth/provider.ts (hunk 2 above)
Comment (kevinszuchet) [reply src/eth/provider.ts:27]: 🙏

File: src/eth/provider.ts (hunk 1 above)
Comment (kevinszuchet) [reply src/eth/provider.ts:4]: My bad 🙏

**Review by kevinszuchet — APPROVED** [review]

**Pull request merged**